doc/sphinx/requirements.txt

```diff
@@ -1,5 +1,6 @@
 sphinx>=3.0
 breathe>=4.17
+markdown==3.0.1
```

Comment:
Was `3.2.1` the one that passed?

Reply:
Good catch, thanks. 3.0.1 is what I had locally. 3.2.1 is working locally as well.